unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Tests/WebVideoPlayerTests.cs

```diff
@@ -0,0 +1,213 @@
+using DCL.Components.Video.Plugin;
+using NSubstitute;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Tests
+{
+    public class WebVideoPlayerTests
```

Comment:
We are moving to the `Should...When` naming convention for tests. New test suites should abide to that convention if possible.

Reply:
Got it, will rename the class for now.